engine/merge_accessors.ml

```diff
@@ -0,0 +1,54 @@
+open Ast
+
```

Comment:
Note: you could `open Target_sym_engine` here to avoid the explicit qualifications below. (But I'm fine with the explicit qualifications, I'm happy to have less things in the global scope.)